pages/eap/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Participate in the Kotlin Early Access Preview
+main_nav_id: eap
+---
+
+# Participate in the Kotlin Early Access Preview (EAP)
+
+You can participate in the Kotlin Early Access Preview (EAP) to try out the latest Kotlin features before they are released.
+
+We ship a few EAP builds before every Kotlin incremental release (_1.x.y_) and a few Milestone (M) builds before every feature release (_1.x_). 
+
+We’ll be very thankful if you find and report bugs to our issue tracker [YouTrack](https://kotl.in/issue). 
+It is very likely that we’ll be able to fix them before the final release, which means you won’t need to wait until the next Kotlin release for your issues to be addressed. 
+
+By participating in the Early Access Preview and reporting bugs, you contribute to Kotlin and help us make it better 
+for everyone in [the growing Kotlin community](https://kotlinlang.org/community/). We appreciate your help a lot! 
```

Comment:
Direct link to kotlinlang.org -- seems like it should be relative path

Reply:
Fixed